elements/lisk-transaction-pool/src/transaction_pool.ts

```diff
@@ -310,19 +308,23 @@ export class TransactionPool {
 
 	// eslint-disable-next-line class-methods-use-this
 	private _calculateFeePriority(trx: Transaction): bigint {
-		return (trx.fee - trx.minFee) / BigInt(trx.getBytes().length);
+		return (trx.fee - this._calculateMinFee(trx)) / BigInt(trx.getBytes().length);
 	}
 
-	// eslint-disable-next-line class-methods-use-this
-	private _getStatus(txResponse: TransactionResponse): TransactionStatus {
-		if (txResponse.status === Status.OK) {
-			debug('Received PROCESSABLE transaction');
+	private _calculateMinFee(trx: Transaction): bigint {
+		const baseFee = this._baseFees.filter(
```

Comment:
`find` and if `undefined` set to `BigInt(0)` would be better in case baseFee will not be  present